app/views/content/train-to-be-a-teacher/if-your-application-is-unsuccessful.md

```diff
@@ -0,0 +1,79 @@
+promo_content:
+    - content/train-to-be-a-teacher/promos/adviser-promo-apply
+---
+
+Try not to worry - you can reapply for up to 4 courses at a time, as many times as you want. Many people are successful the second or third time around.
+
+It’s worth looking at the feedback the course provider has given to understand why your application was not accepted. This will help you to decide on your next steps.
+
+## Why was my application unsuccessful?
+
+There could be a number of reasons your application was unsuccessful:
+
+### The course is full
+
+Unfortunately, popular courses do fill up quickly. You can [search for other courses](https://www.gov.uk/find-postgraduate-teacher-training-courses) with a different provider if the one you applied for is now full.
+
+### You do not have the GCSEs at minimum grade 4 (C), or equivalent
+
+To train to teach in primary and secondary schools in England, you’ll need:
+
+* GCSEs at grade 4 (C) or above in English and maths (and science if you want to teach primary)
+
+You could [find another course](https://www.gov.uk/find-postgraduate-teacher-training-courses) with a training provider that will accept an equivalency test. You can also take your GCSE exams at any age if you do not have them or you can retake them to improve your grades.
+
+### You do not have a bachelor’s degree or equivalent
+
+You’ll need a bachelor's degree or equivalent to train to teach. If you do not have one, you can still train to teach by doing an [undergraduate degree in teacher training](/train-to-be-a-teacher/if-you-dont-have-a-degree) or become a [further education teacher](/become-a-further-education-teacher).
+
+### The provider could not verify your qualifications
+
+To help show your qualifications, you can [get a certified statement of your exam results](https://www.gov.uk/replacement-exam-certificate) if you did your GCSEs and A levels in the UK. You should also be able to request a copy of your degree certificate from the organisation where you studied in the UK.
+
+If you got your qualifications outside the UK, you should be able to request copies of your school results and degree from the places where you did them.
+
+### Lack of subject knowledge
+
+You may be able to do a [subject knowledge enhancement (SKE) course](/train-to-be-a-teacher/subject-knowledge-enhancement) to top up your subject knowledge. You could also find a course in a different subject.
+
+### Lack of teaching or safeguarding knowledge
+
+It may help to [get some school experience](/train-to-be-a-teacher/get-school-experience) so you can find out what it’s like to be a teacher and improve your knowledge and skills.
+
+### No visa sponsorship
+
+Find out more about [how visa sponsorships work](/non-uk-teachers/train-to-teach-in-england-as-an-international-student) if you want to train to teach in the UK. You may also find another training provider that will sponsor a visa by filtering search results on ‘visa sponsorship’ when you search for courses.
```

Comment:
```suggestion
Find out more about [how visa sponsorships work](/non-uk-teachers/train-to-teach-in-england-as-an-international-student) if you want to train to teach in the UK. You may also find another training provider that will sponsor a visa by searching for courses and then filtering by ‘Only show courses with visa sponsorship’.
```

Would it be clearer to flip this around - ie search first, then filter?

I've also put the full filter name - but change back if you feel this is not necessary